tests/integration/seed/seedsVerification.test.js

```diff
@@ -767,6 +790,13 @@ describe('SEEDS VERIFICATION', () => {
           expect(onlyIntraHoldingCoursesHaveHolding).toBeTruthy();
         });
 
+        it('should pass if every holding exists', () => {
```

Comment:
En lisant l'intutlé , on dirait qu'on fait un test sur l'existance de la holding. On pourrait plutot mettre qqchose du style:  "should pass if holding is defined for every INTRA_HOLDING course", qu'en penses-tu ?

Reply:
c'est ce que je fais, je vérifie que la holding n'est pas new ObjectId

Reply:
Ah oui pardon ! J'avais oublié qu'on populait le champ holding dans la def de courseList ;)